android/viewsets/lexicon/src/main/java/com/liferay/mobile/screens/viewsets/lexicon/ddl/form/fields/DDLFieldTextView.java

```diff
@@ -38,6 +42,15 @@ public DDLFieldTextView(Context context, AttributeSet attributes, int defaultSty
 
     @Override
     public void onPostValidation(boolean valid) {
-        FormViewUtil.setupTextFieldLayout(getContext(), valid, labelTextView, textEditText);
+        FormViewUtil.setupBackground(getContext(), valid, textEditText);
+
+        boolean isRepeatableField = getParent() instanceof DDMFieldRepeatableItemView;
+
+        if (!isRepeatableField) {
+            View errorView = findViewById(R.id.error_view);
+            if (errorView != null) {
+                errorView.setVisibility(valid ? GONE : VISIBLE);
+            }
```

Comment:
``` 
View errorView = findViewById(R.id.error_view);
if (errorView != null) {
    errorView.setVisibility(valid ? GONE : VISIBLE);
}
```
This code can be extracted to a function because its called in several places

Reply:
Yeah, I extracted in a Util function. Thanks!